linebot/api.py

```diff
@@ -1034,6 +1034,88 @@ def get_insight_message_event(self, request_id, timeout=None):
+        :param str client_assertion_type: `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`.
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :rtype: :py:class:`linebot.models.responses.IssueChannelTokenResponse`
+        :return: IssueChannelTokenResponse instance
+        """
+        response = self._post(
+            '/oauth2/v2.1/token',
+            data={
+                'grant_type': grant_type,
+                'client_assertion_type': client_assertion_type,
+                'client_assertion': client_assertion,
+            },
+            headers={'Content-Type': 'application/x-www-form-urlencoded'},
+            timeout=timeout
+        )
+
+        return IssueChannelTokenResponse.new_from_json_dict(response.json)
+
+    def revoke_channel_access_token_v2_1(self, client_id,
+                                         client_secret, access_token,
+                                         timeout=None):
+        """Revokes a channel access token v2.1.
+
+        https://developers.line.biz/en/reference/messaging-api/#revoke-channel-access-token-v2-1
+
+        :param str client_id: Client id.
+        :param str client_secret: Channel secret.
+        :param str access_token: Channel access token.
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        """
+        self._post(
+            '/oauth2/v2.1/revoke',
+            data={'client_id': client_id,
+                  'client_secret': client_secret,
+                  'access_token': access_token},
+            timeout=timeout
+        )
+
+    def get_channel_access_tokens_v2_1(
+            self, client_assertion,
+            client_assertion_type='urn:ietf:params:oauth:client-assertion-type:jwt-bearer',
+            timeout=None):
+        """Get issued channel access tokens v2.1.
+
+        https://developers.line.biz/en/reference/messaging-api/#get-issued-channel-access-tokens-v2-1
+
+        :param str client_assertion: Clint assertion.
+        :param str client_assertion_type: `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`.
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :rtype: :py:class:`linebot.models.responses.ChannelAccessTokens`
+        :return: ChannelAccessTokens instance
```

Comment:
[imo]
I think that the return description is unnecessary for consistency with other descriptions. (`rtype` alone is enough)